website/content/docs/internals/telemetry/enable-telemetry.mdx

```diff
@@ -57,29 +57,29 @@ telemetry {
   num_lease_metrics_buckets = 168
   add_lease_metrics_namespace_labels = false
   filter_default = true
-  
+
   statsite_address = "mycompany.statsite:8125"
 }
 ```
 
-<Tip heading="Use a prefix filter to reduce the volume of metrics you receive">
+<Tip>
 
   Many metrics solutions charge by the metric. You can set `filter_default` to
   false and use the `prefix_filter` parameter to include and exclude specific
   values based on metric name to avoid paying for irrelevant information.
-  
-  For example, to limit your telemetry to the core token metrics plus the number
-  of leases set to expire:
-
-  ```hcl
-  telemetry {
-  filter_default = false
-    prefix_filter = ["+vault.token", "-vault.expire", "+vault.expire.num_leases"]
-  }
-  ```
 
 </Tip>
 
+For example, to limit your telemetry to the core token metrics plus the number
```

Comment:
Moving the example out of the tip doesn't really make sense. If the visual bug is unacceptable, we should remove the tip altogether and simple include it as part of the main content body rather than as an aside.